glean-core/tests/common/mod.rs

```diff
@@ -12,6 +12,20 @@ use chrono::offset::TimeZone;
 use iso8601;
 use iso8601::Date::YMD;
 
+use ctor::ctor;
+
+/// Initialize the logger for all tests without individual tests requiring to call the init code.
+/// Log ouptut can be controlled via the environment variable `RUST_LOG` for the `glean_core` crate,
+/// e.g.:
+///
+/// ```
+/// export RUST_LOG=glean_core=debug
```

Comment:
Should this bit of information go in #117 ? If that's merged before, can you please add it to the document in this PR?